src/architecture/messaging/_UnitTest/test_MessagingInclude.py

```diff
@@ -0,0 +1,40 @@
+#
+#  ISC License
+#
+#  Copyright (c) 2024, Autonomous Vehicle Systems Lab, University of Colorado at Boulder
+#
+#  Permission to use, copy, modify, and/or distribute this software for any
+#  purpose with or without fee is hereby granted, provided that the above
+#  copyright notice and this permission notice appear in all copies.
+#
+#  THE SOFTWARE IS PROVIDED "AS IS" AND THE AUTHOR DISCLAIMS ALL WARRANTIES
+#  WITH REGARD TO THIS SOFTWARE INCLUDING ALL IMPLIED WARRANTIES OF
+#  MERCHANTABILITY AND FITNESS. IN NO EVENT SHALL THE AUTHOR BE LIABLE FOR
+#  ANY SPECIAL, DIRECT, INDIRECT, OR CONSEQUENTIAL DAMAGES OR ANY DAMAGES
+#  WHATSOEVER RESULTING FROM LOSS OF USE, DATA OR PROFITS, WHETHER IN AN
+#  ACTION OF CONTRACT, NEGLIGENCE OR OTHER TORTIOUS ACTION, ARISING OUT OF
+#  OR IN CONNECTION WITH THE USE OR PERFORMANCE OF THIS SOFTWARE.
+#
+
+from Basilisk.moduleTemplates import cppModuleTemplate
+
+def test_messagingInclude(show_plots):
```

Comment:
Agree with moving to `numpy.testing`. However, since this tests is only checking that a method does not raise an error, you probably don't even need `numpy.testing`. If we want to remove `testFailCount` and `testMessages`, we might as well:
```
    try:
        dataLog = module.dataOutMsg.recorder()
    except:
        assert False, "messaging inclusion test failed"
```
or even just let the error be raised and caught by `pytest`:
```
    dataLog = module.dataOutMsg.recorder()
```

Reply:
I'll play with these ideas to simplify the testing.  In the end, this has to work and be reliable in testing.  Lots of ways to do this ;-)